src/Database/Query.php

```diff
@@ -889,6 +889,50 @@ public function where($conditions = null, $types = [], $overwrite = false)
         return $this;
     }
 
+    /**
+     * Adds an IN condition or set of conditions to be used in the WHERE clause for this
+     * query.
+     *
+     * This method does allow empty inputs in contrast to where() if you set
+     * $allowEmpty to true.
+     * Be careful about using it without proper sanity checks.
+     *
+     * @param string $field Field
+     * @param array $values Array of values
+     * @param bool $allowEmpty Allow empty array
+     * @return $this
+     */
+    public function whereIn($field, array $values, $allowEmpty = false)
```

Comment:
By type hinting `$values` as array you are prevent this method to be used for cases where value is a string. Is that intentional?  Because for e.g. `->where(['title IN' => 'foo'])` is valid usage which generate SQL fragment `title in ('foo')`.

Reply:
I would say it is intentional and desired, a string is not a list of values.

Reply:
Casting beforehand is eaay. And this way no accidental usage of wrong types.

Reply:
Works for me. Just wanted to confirm.